packages/office-ui-fabric-react/src/components/Slider/Slider.styles.ts

```diff
@@ -301,24 +304,57 @@ export const getStyles = (props: ISliderStyleProps): ISliderStyles => {
             transform: 'translateY(-6px)'
           }
     ],
+    regularLabel: [
```

Comment:
Just a question, why are these called `regularLabel` and `regularTick`? Are there non-regular versions coming up?

Reply:
Regular Tick was called that because there is already a ZeroTick so I just thought this is like the regular one so I called it regularTick
RegularLabel was called that because there's already a valueLabel so I just thought that this is the regular labels